src/util/type.ts

```diff
@@ -27,6 +27,7 @@ export type ISettingsForm = {
   weekHourStart?: number
   weekHourEnd?: number
   ignoreTag?: string
+  overdueCutoffDays?: number
```

Comment:
```suggestion
  overdueCutoffDays?: number | null
```
It appears that overdueCutoffDays may still be null.

Reply:
Would you mind explaining a bit more about your suggestion? I intended to set it as `number | undefined` so it aligns with the others. When it is undefined it means this setting is unset. Using `null` as the default value is also a good idea but I think it is better to be `number | null` in this case and become a required property.

Reply:
Based on the code below, I can see that when the user selects 'Never', the value of `overdueCutoffDays` is set to null. From this observation, it appears that `overdueCutoffDays` can have three types of values:
1. `undefined`, which indicates that the user has not made a selection.
2. `null`, which represents the user's choice of 'Never'.
3. `number`, which corresponds to the user's selection of a specific number of days.


<img width="989" alt="image" src="https://user-images.githubusercontent.com/17695989/229279702-cb2dc2ad-d673-46e5-91d5-fef847ca1ec2.png">